src/main/java/com/cricket/project/dto/MatchDto.java

```diff
@@ -0,0 +1,16 @@
+package com.cricket.project.dto;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.util.Date;
+
+@Data
+public class MatchDto {
```

Comment:
while creating the DTOs we should always use `@JsonIgnoreProperties(ignoreUnknown = true)` so that if someone passes anything extra in the JSON the code doesn't break.

Reply:
ok noted.